src/main/java/org/sunbird/assessment/service/AssessmentServiceImpl.java

```diff
@@ -40,6 +54,15 @@
 	@Autowired
 	AssessmentUtilService assessUtilServ;
 
+	@Autowired
+	RestTemplate restTemplate;
```

Comment:
Do not use restTemplate and call REST API from service classes.
Use OutboundRequestHandlerServiceImpl to raise a REST request